src/plugins/dataflow/components/ui/dataflow-rateselector-playback.tsx

```diff
@@ -114,7 +171,7 @@ export const RateSelectorOrPlayBack = (props: IRateSelectorProps) => {
             </select> :
             numNodes > 0 &&
             <div className="countdown-timer">
-             {`${formattedTime} / ${totalTimeFormatted}`}
+              {programRecordState === 1 ? `${formattedTime} / ${totalTimeFormatted}` : `${playBackFormattedTime}/${formattedTime}`}
```

Comment:
This line is too long.

Reply:
Split it into this and merging. 
<img width="780" alt="image" src="https://user-images.githubusercontent.com/521934/230491662-d607fc01-c64a-469f-af16-8e469fa8ec8d.png">